build.sbt

```diff
@@ -31,7 +31,8 @@ licenses := Seq("Apache-2.0" -> url("http://opensource.org/licenses/Apache-2.0")
 spAppendScalaVersion := true
 
 // Add Spark components this package depends on, e.g, "mllib", ....
-sparkComponents ++= Seq("graphx", "sql")
+// We need mllib for sparse vector encoded parallel personalized PageRank
+sparkComponents ++= Seq("graphx", "sql", "mllib")
```

Comment:
It seems that `GraphFrames` has to register the sparse vector column type as as `UserDefinedType`. The corresponding definition is only available in `mllib` https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/linalg/SQLDataTypes.scala